apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TapToToneActivity.java

```diff
@@ -116,6 +117,8 @@ private void updateButtons(boolean running) {
 
     void trigger() {
         if (mTapToToneTester.isArmed()) {
+            Timestamp timestamp = new Timestamp(System.currentTimeMillis());
```

Comment:
The mAudioOutTester.trigger() call is very fast so please put this new code after it.
That way the log() will not contribute to the measured latency.

Reply:
Thanks!